### PR TITLE
SDK docs

### PR DESCRIPTION
**Pull request by krofax**
Documentation for Java, Python, and Go SDK's.

### CODE REVIEW

File: docs/api/go/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Get started
```
Comment (uF4No) [inline docs/api/go/getting-started.md:1]: This title should be "Getting started to match the other pages"

File: docs/api/go/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+```go 
+
+get github.com/zksync-sdk/zksync2-go
+
+```
+
+## How to init main components
+Using these three instances `ZkSync Provider`, `EthereumProvider` and `Wallet`, you can perform all basic actions with ZkSync network, just explore their public methods and package helpers.
+
+A few examples are below:
```
Comment (uF4No) [inline docs/api/go/getting-started.md:28]: Avoid using passive, should be, "Here are a few examples" or something similar

File: docs/api/go/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+## Depositing funds
+
+```go
+
```
Comment (uF4No) [inline docs/api/go/getting-started.md:33]: I'm not familiar with Go but, should we have to include some imports on the top of this script?

File: docs/api/java/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,831 @@
+# Getting started
+
```
Comment (uF4No) [inline docs/api/java/getting-started.md:2]: Missing intro line like in the Go page "In this guide we will demonstrate how to:"

File: docs/api/go/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+In this guide we will demonstrate how to:
+
+1. Connect to the zkSync network.
+2. Deposit assets from Ethereum into zkSync.
+3. Check Balance 
+4. Transfer funds (ERC20 tokens) into the zksync network
+5. Withdraw funds (Native coins)
+
+## Prerequisite
+This guide assumes that you are familiar with the basics of go and you have it configured on your local machine.
```
Comment (uF4No) [inline docs/api/go/getting-started.md:12]: "you are familiar with the Go programming language", including a link to the Go documentation

File: docs/api/java/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,831 @@
...
+1. Connect to the zksync network.
+2. Deposit assets from Ethereum into zksync.
+3. Check Balance 
+4. Transfer funds (ERC20 tokens) into the zksync network
+5. Withdraw funds (Native coins)
+6. Deploy contract with create method
+
+## Prerequisite
+This guide assumes that you are familiar with the basics of java and you have java configured on your local machine.
```
Comment (uF4No) [inline docs/api/java/getting-started.md:11]: "you are familiar with the basics of Java", including a link to the Java documentation. Use capitals

File: docs/api/java/getting-started.md [hunk 7]
```diff
@@ -0,0 +1,831 @@
...
+
+##  Installation
+For connecting zksync java SDK, just add the following dependency to your build file.
```
Comment (uF4No) [inline docs/api/java/getting-started.md:14]: "zkSync" and "Java", let's keep names like this throughout the whole document

File: docs/api/python/getting-started.md [hunk 8]
```diff
@@ -0,0 +1,580 @@
...
+1. Connect to the zkSync network.
+2. Deposit assets from Ethereum into zkSync.
+3. Check Balance 
+4. Transfer funds (ERC20 tokens) into the zksync network
+5. Withdraw funds (Native coins)
+6. Deploy contract with create method
+7. Deploy contract with create2 method
+
+## Prerequisite
+This guide assumes that you are familiar with the basics of python and you have python configured on your local machine.
```
Comment (uF4No) [inline docs/api/python/getting-started.md:14]: Python with capital and link to its docs

File: docs/api/python/getting-started.md [hunk 9]
```diff
@@ -0,0 +1,580 @@
...
+        value += x;
+    }
+
+    function get() public view returns (uint256) {
+        return value;
+    }
+}
+
+```
+> It must be compiled by zkSync compiler only!
```
Comment (uF4No) [inline docs/api/python/getting-started.md:349]: Maybe add a link to a section in our docs where we mention how to compile contracts

File: docs/api/python/getting-started.md [hunk 10]
```diff
@@ -0,0 +1,580 @@
...
+
+After compilation there must be 2 files with the:
+
+- contract binary representation
+- contract ABI in JSON format
+
+### Step 2: Deploy the contract
+
+To deploy the contract, contract ABI is needed for calling its methods in the standard `web3` way.
```
Comment (uF4No) [inline docs/api/python/getting-started.md:358]: Do we need to mention this in this section of the docs that is just about deploying the contract?

**Comment by uF4No** [comment]
As Ramón mentioned, if the SDKs are not fully tested, I'd consider making this a WIP PR. 
Also, I think once we add these new sections, we'd need to update the [Overview page](https://aqwzx-zksync-v2-docs--pr112-sdk-docs-yj3g7bfj.web.app/api/#sdk)

File: docs/api/python/getting-started.md [hunk 11]
```diff
@@ -0,0 +1,594 @@
...
+
+URL_TO_ETH_NETWORK = "https://goerli.infura.io/v3/25be7ab42c414680a5f89297f8a11a4d"
+ZKSYNC_NETWORK_URL = "https://zkSync2-testnet.zkSync.dev"
+
+eth_web3 = Web3(Web3.HTTPProvider(URL_TO_ETH_NETWORK))
+zkSync_web3 = zkSyncBuilder.build(ZKSYNC_NETWORK_URL)
+
+```
+
+The SDK supports the goerli, and zkSync testnet networks.
```
Comment (ramonatmatterlabs) [inline docs/api/python/getting-started.md:45]: I don't think this is true... should support mainnet... please confirm with Max

**Review by ramonatmatterlabs — CHANGES_REQUESTED** [review]

File: docs/.vuepress/config.js [hunk 12]
```diff
@@ -159,6 +159,27 @@ module.exports = {
...
+        {
+          title: "Python SDK", // required
+          path: "/api/python", // optional, which should be a absolute path.
+          collapsable: false, // optional, defaults to true
+          sidebarDepth: 1, // optional, defaults to 1
+          children: ["/api/python/getting-started"],
+        },
+        {
+          title: "Java SDK", // required
+          path: "/api/java", // optional, which should be a absolute path.
```
Comment (uF4No) [inline docs/.vuepress/config.js:171]: broken link in menu. Add "/gettting-started" to path

File: docs/.vuepress/config.js [hunk 13]
```diff
@@ -159,6 +159,27 @@ module.exports = {
             "/api/js/front-end",
           ],
         },
+        {
+          title: "Python SDK", // required
+          path: "/api/python", // optional, which should be a absolute path.
```
Comment (uF4No) [inline docs/.vuepress/config.js:164]: broken link in menu. Add "/gettting-started" to path

File: docs/.vuepress/config.js [hunk 14]
```diff
@@ -159,6 +159,27 @@ module.exports = {
...
+        {
+          title: "Java SDK", // required
+          path: "/api/java", // optional, which should be a absolute path.
+          collapsable: false, // optional, defaults to true
+          sidebarDepth: 1, // optional, defaults to 1
+          children: ["/api/java/getting-started"],
+        },
+        {
+          title: "GO SDK", // required
+          path: "/api/go", // optional, which should be a absolute path.
```
Comment (uF4No) [inline docs/.vuepress/config.js:178]: broken link in menu. Add "/gettting-started" to path

File: docs/api/java/getting-started.md [hunk 15]
```diff
@@ -0,0 +1,835 @@
...
+2. Deposit assets from Ethereum into zkSync.
+3. Check Balance 
+4. Transfer funds (ERC20 tokens) into the zkSync network
+5. Withdraw funds (Native coins)
+
+## Prerequisite
+This guide assumes that you are familiar with the basics of [Java programming](https://docs.oracle.com/en/java/) language.
+
+##  Installation
+For connecting zkSync Java SDK, just add the following dependency to your build file.
```
[the remark this answers is not in the document]
Comment (krofax) [reply docs/api/java/getting-started.md:15]: It's important to specify "build file", I'd vote that this remains.

**Comment by ramonatmatterlabs** [comment]
Build file only makes sense for Android with gradle no? On maven projects
for example dependencies lives on a different place

File: docs/api/go/getting-started.md [hunk 16]
```diff
@@ -0,0 +1,209 @@
...
+        panic(err)
+    }
+    fmt.Println("Deployed address", addr.String())
+}
+
+```
+
+## Execute smart contract (basic case)
+
+In order to interact with smart contract, the SDK needs to know the contract address you want to interact with, as well as its ABI. For that, you need to use ABI.Pack() [method]("https://github.com/ethereum/go-ethereum/accounts/abi") to load the ABI of your contract, or encode the calldata to execute function and its parameters.
```
Comment (uF4No) [inline docs/api/go/getting-started.md:181]: Remove double quotes from link, it fails

File: docs/api/python/getting-started.md [hunk 17]
```diff
@@ -0,0 +1,593 @@
...
+To start using this SDK, you just need to pass in a provider configuration.
+Once you have all the necessary dependencies, you can follow the following setup steps to get started with the SDK.
+To interact with zkSync network users need to connect the endpoint of the operator node.
+
+```py
+
+from web3 import Web3
+from zkSync2.module.module_builder import zkSyncBuilder
+
+URL_TO_ETH_NETWORK = "https://goerli.infura.io/v3/25be7ab42c414680a5f89297f8a11a4d"
```
Comment (uF4No) [inline docs/api/python/getting-started.md:37]: Remove Infura URL with API key and replace with "GOERLI_HTTPS_RPC"

File: docs/api/python/getting-started.md [hunk 18]
```diff
@@ -0,0 +1,593 @@
...
+ZKSYNC_NETWORK_URL = "https://zkSync2-testnet.zkSync.dev"
```
Comment (uF4No) [inline docs/api/python/getting-started.md:38]: url should be all lowercase

File: docs/api/go/getting-started.md [hunk 19]
```diff
@@ -0,0 +1,209 @@
...
+
+package main
+
+import (
+    "github.com/ethereum/go-ethereum/rpc"
+    "github.com/zksync-sdk/zksync2-go"
+)
+
+// first, init Ethereum Signer, from your mnemonic
+es, err := zksync2.NewEthSignerFromMnemonic("<mnemonic words>", zksync2.ZkSyncChainIdMainnet)
```
Comment (uF4No) [inline docs/api/go/getting-started.md:43]: in general, I think we should improve variable names in all code samples. `es` stands for ethereum signer, so let's name it that way. Same with `zp` (zksync provider) or `w` (wallet)

**Comment by uF4No** [comment]
Can we add a warning on all three new pages mentioning that this section of the docs is still in progress and that it'll be updated with more detailed information?

**Review by githubdoramon — APPROVED** [review]

**Review by ramonatmatterlabs — APPROVED** [review]

**Pull request merged**